.envrc

```diff
@@ -1,6 +1,5 @@
-#!/usr/bin/env bash
+# shellcheck shell=bash
```

Comment:
What happens if you don't have shellcheck, I wonder if removing shebang works?

Reply:
> What happens if you don't have shellcheck

This is a comment, so shellcheck isn't called. This is something shellcheck looks for.

> I wonder if removing shebang works?

We _used_ to have a shebang here that was added to specifically fix the shellcheck hook. It's not needed for direnv to run the file.

Reply:
Looking back at this, maybe we should just add `#!/usr/bin/env bash` to all of our `.envrc` files?

There doesn't seem to be any harm in doing so and I don't like having a linter-specific comment in all the files we ship.